definitions/install.yaml

```diff
@@ -738,6 +738,7 @@ spec:
           - "--filterK8Resources=[Event,*,*][*,kube-system,*][*,kube-public,*][*,kube-node-lease,*][Node,*,*][APIService,*,*][TokenReview,*,*][SubjectAccessReview,*,*][*,kyverno,*][Binding,*,*][ReplicaSet,*,*]"
           # customize webhook timeout
           #- "--webhooktimeout=4"
+          - "-v=6"
```

Comment:
I think we want to keep the default log level at 2....is there a reason to increase?